README.md

```diff
@@ -0,0 +1,37 @@
+# Link All The Languages!
+-------------------------
+
+A big bundle of different languages all being invoked from a C program.
+
+## Why?
+
+We want to see if we can!
+
+## Getting Started
+This project has two build systems:
+ - A GNU Makefile
+ - A custom Python 3 script
+If you choose to use the GNU Makefile, you must install all of the language toolchains (listed below)
+used in this project.
```

Comment:
I'd actually like to start deprecating the Makefile in favor of your Python script.

Reply:
I've made a [tracking issue](https://github.com/cbpudding/link-all-the-languages/issues/13) for this. Added notice about that in the README.